.github/workflows/cron.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           node-version-file: .node-version
           registry-url: https://registry.npmjs.org
       - name: Install Yarn
-        run: curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.13.0
+        run: curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.22.22
```

Comment:
I had some weird issues where 1.13 was not resolving dependencies correctly and the builds would always fail on changes to the yarn.lock file.